segwit.asciidoc

```diff
@@ -10,13 +10,13 @@ Will be merged later into chapter 6 or 7, as the book is reorganized
 
 Segregated Witness (segwit) is an upgrade to the bitcoin consensus rules and network protocol, scheduled for implementation in the second half of 2016.
 
-In cryptography, the term "witness" is used to describe a solution to a cryptographic puzzle. In bitcoin terms, the witness satisfies a cryptographic condition placed on a Unspent Transaction Output (UTXO).
+In cryptography, the term “witness” is used to describe a solution to a cryptographic puzzle. In bitcoin terms, the witness satisfies a cryptographic condition placed on an Unspent Transaction Output (UTXO).
```

Comment:
@elias19r Could you please revise this PR and remove the smart quotes? Dumb quotes are already in use in many places throughout the book.